detail.html

```diff
@@ -0,0 +1,34 @@
+<%inherit file="app_base.htm" />
+
+<%block name ="content_top">
+</%block>
+
+<div id = "pagestuff">
+  <%block name = "content_middle">
+    <div id = "small_imgs">
+    <ul>
+      %for pic in product.image_urls():
+        <li class = "hover_item"><img src = "${pic}" alt = "main picture for ${product_name}"/></li>
+      %endfor
+    </ul>
+    </div>
+    <div id = "big_img">
+      <img src="${product.image_url()}" alt = "alternate picture for ${product_name}"/>
```

Comment:
I like the way you made the alt tag unique for each image. If only there was a way to describe the picture.

Reply:
Yeah, i wish haha